Prox/Prox/PlaceDetails/PlaceDetailsDescriptionView.swift

```diff
@@ -55,6 +56,8 @@ class PlaceDetailsDescriptionView: UIView {
          icon: UIImage?,
          horizontalMargin: CGFloat) {
         self.horizontalMargin = horizontalMargin
+        toggleEventType = labelText == "Yelp top review" ? AnalyticsEvent.YELP_TOGGLE : AnalyticsEvent.WIKIPEDIA_TOGGLE
```

Comment:
This should not be in the `PlaceDetailsDescriptionView`. It's too fragile. If we were to change the text of the label in `PlaceDetailsCardView` without changing the test here then this analytic wouldn't send.

I suggest adding a variable of `var analyticsEvent: AnalyticsEvent` and set it when we create the description view insode `PlaceDetailsCardView` to the correct value, then use that value instead.